recipe/meta.yaml

```diff
@@ -63,62 +59,75 @@ source:
 
+
 build:
-  number: 1
+  number: 0
+  add_pip_as_python_dependency: false
   # Windows has issues updating python if conda is using files itself.
   # Copy rather than link.
   no_link:
     - DLLs/_ctypes.pyd  # [win]
-    - bin/python3.7  # [not win]
-  script_env:
-    - python_branding
-  skip: true  # [win and vc != 14]
+  # We delete the shared libraries.
+  ignore_run_exports:   # [unix]
+    - bzip2             # [unix]
+  detect_binary_files_with_prefix: False
+  # match python.org compiler standard
+  skip: True            # [win and vc != 14]
+# Would like to be able to append to the calculated build string here, ping @msarahan, is this possible?
+# Also this messes up hash generation so package ends up without any :-(
+# {% if linkage_nature != '' %}
+#   string: {{ version }}{{ linkage_nature }}
+# {% endif %}
+#  script_env:
+#    - PY_INTERP_LINKAGE_NATURE
+#    - python_branding
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
-    - make                               # [not win]
-    - pkg-config                         # [not win]
-    # - 7za                                # [win]
-    # - curl                               # [win]
+    - make                               # [unix]
+    - pkg-config                         # [unix]
+    - patch                              # [not win]
+    - m2-patch                           # [win]
   host:
-    # - nasm 2.11.06      # [win]
-    - bzip2             # [not win]
-    - sqlite
-    - xz                # [not win]
-    - zlib              # [not win]
-    - openssl
-    - readline          # [not win]
-    - tk                # [not win]
-    - ncurses           # [linux]
-    - libffi            # [not win]
-  run:
-    - bzip2             # [not win]
+    - bzip2                              # [unix]
     - sqlite
-    - xz                # [not win]
-    - zlib              # [not win]
+    - xz                                 # [unix]
+    - zlib                               # [unix]
     - openssl
-    - readline          # [not win]
-    - tk                # [not win]
-    - ncurses           # [linux]
-    - libffi            # [not win]
+    - readline                           # [unix]
+    - tk                                 # [unix]
+    - ncurses                            # [unix]
+    - libffi                             # [unix]
+   # run:
+   # # conda-build bug?
+   #  - ncurses >=6.1,<7.0a0               # [unix]
 
 test:
+  # Because we need to use xvfb-run on linux, from a CDT package.
+  requires:  # [linux]
+    # We are running centos6 software here (on whatever distro you are on)
+    # .. so it best be compatible (you should use centos6 if possible).
+    - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
+    - {{ cdt('xorg-x11-xauth') }}  # [linux]
```

Comment:
These CDT do not exist yet.  They will need to be created.

Reply:
Just `xorg-x11-xauth`. I lost my phone's SIM unlock code (yeah really) so haven't been able to run `c3i` to build it yet. It's ok, I recovered it now.